Gemfile.linters

```diff
@@ -0,0 +1,14 @@
+# All gems required to run linters:
```

Comment:
This file is here because juggling the compatibility of these gems can be difficult (for running linters locally), but I don't want to bring all of their dependencies into the main Gemfile and make it harder for automatic updates.

Reply:
Can you explain in small words how this is used? Do you run the commands on lines 3-5 to install the gems for the linters, or...?

Reply:
I got the idea from ["Is it possible to override gemfile for local development?"](https://stackoverflow.com/questions/48863711/is-it-possible-to-override-gemfile-for-local-development).

- Run `BUNDLE_GEMFILE=Gemfile.linters bundle install` once to install the linter gems + their dependencies + our existing dependencies. Re-run it occasionally to update (bundler will prompt us).
- Run `BUNDLE_GEMFILE=Gemfile.linters bundle exec rubocop <file>` and `BUNDLE_GEMFILE=Gemfile.linters bundle exec erblint <file>` as regularly as we want, to see Rubocop and ERB Lint violations.

Reply:
Makes sense! Should we document that on the [coding standards wiki page](https://github.com/otwcode/otwarchive/wiki/Coding-standards) for folks who want to check that their changes meet our style guidelines before submitting PRs?

Reply:
Yep, replacing this section specifically: https://github.com/otwcode/otwarchive/wiki/Coding-standards#Previewing_Hound_comments.

The downside of running these commands is that people will see all existing violations in a file instead of just the ones on the lines they added/edited (like how Hound works on GitHub). I think https://github.com/AtakamaLLC/lint-diffs looks promising, but that's for later.

Reply:
Wiki updated.